index.js

```diff
@@ -146,7 +146,7 @@ function compile(entrypoint, cache, options) {
     const inputs = new Set();
 
     const b = browserify(entrypoint, {
-      basedir: (path.extname(entrypoint) === '.js') ? path.dirname(entrypoint) : entrypoint,
+      basedir: (path.extname(entrypoint) === '.js') ? process.cwd() : entrypoint,
```

Comment:
Let's use `process.cwd()` regardless here – the original logic was just trying to get the directory name of either the file, if you pointed it at that, or, the directory if you pointed it at a directory (which would then need to have an `index.js` file as the entrypoint).